src/matrix.ts

```diff
@@ -51,8 +51,14 @@ export {
     createNewMatrixCall,
     setAudioInput as setMatrixCallAudioInput,
     setVideoInput as setMatrixCallVideoInput,
+    CallType,
 } from "./webrtc/call";
 
+// TODO: This export is temporary and is only used for the local call feed for conference calls
+// Ideally conference calls will become a first-class concept and we will have a local call feed with
+// a lifecycle that matches the conference call, not individual calls to members.
+export { CallFeed } from "./webrtc/callFeed";
```

Comment:
Note for myself. Now that GroupCall has it's own `localParticipant` `CallFeed` we don't need to export this.